packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2013,7 +2017,13 @@ export default mixins(
 
 			this.historyStore.startRecordingUndo();
 
-			const newNodeData = await this.injectNode(nodeTypeName, options, showDetail, trackHistory);
+			const newNodeData = await this.injectNode(
+				nodeTypeName,
+				options,
+				showDetail,
+				trackHistory,
+				isAutoAdd,
```

Comment:
am I missing something? is this `isAutoAdd` needed?

Reply:
It's only for tracking. We want to distinguish if node(pretty much just Manual Trigger at this point) was added by the user or if it was inserted because there's no trigger on the canvas